std/math.d

```diff
@@ -4270,10 +4270,13 @@ struct IeeeFlags
 private:
     // The x87 FPU status register is 16 bits.
     // The Pentium SSE2 status register is 32 bits.
+    // PowerPC FPSCR is a 32-bit register.
+    // ARM FPSCR is a 32bit register
+    // SPARC FSR is a 32bit register (64 bits for SPARC 7 & 8, but high bits are uninteresting).
     uint flags;
+
     version (X86_Any)
```

Comment:
It already is in druntime, but as stated before, the Microsoft headers don't appear to match the hardware.  And so the duplication is an unfortunate evil.

Maybe there's a rational explanation like Microsoft get and set feflags differently, or mask them somehow to translate to the "correct" values.  Correct being in quotation marks.